From fd3afc9b0508c4d1594db0086343a7ecbb03825d Mon Sep 17 00:00:00 2001
From: chenxing <chenxing@rock-chips.com>
Date: Wed, 23 Jan 2013 10:00:04 +0800
Subject: [PATCH] rk30: export
 dvfs_get_freq_volt_table/dvfs_set_freq_volt_table functions

---
 arch/arm/plat-rk/dvfs.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-rk/dvfs.c b/arch/arm/plat-rk/dvfs.c
index 48de782dd988..523e72a1458a 100755
--- a/arch/arm/plat-rk/dvfs.c
+++ b/arch/arm/plat-rk/dvfs.c
@@ -253,7 +253,7 @@ struct cpufreq_frequency_table *dvfs_get_freq_volt_table(struct clk *clk)
 	mutex_unlock(&mutex);
 	return table;
 }
-
+EXPORT_SYMBOL(dvfs_get_freq_volt_table);
 
 int dvfs_set_freq_volt_table(struct clk *clk, struct cpufreq_frequency_table *table)
 {
@@ -294,6 +294,7 @@ int dvfs_set_freq_volt_table(struct clk *clk, struct cpufreq_frequency_table *ta
 	mutex_unlock(&mutex);
 	return 0;
 }
+EXPORT_SYMBOL(dvfs_set_freq_volt_table);
 
 int dvfs_set_depend_table(struct clk *clk, char *vd_name, struct cpufreq_frequency_table *table)
 {
@@ -984,7 +985,7 @@ void avs_init_val_get(int index, int vol, char *s)
 }
 int avs_set_scal_val(u8 avs_base)
 {
-
+	return 0;
 }
 
 /*************************interface to get avs value and dvfs tree*************************/
-- 
2.35.3

